### PR TITLE
Updated Documentation with Zen and Matrix view

### PR DESCRIPTION
**Pull request by vabene1111**
## Proposed changes

Update the user documentation to include mentions of zen and matrix mode. These changes were requested in issues #5073  and #3345 which are fixed by this PR

## Checklist

- [x] Lint and unit tests pass locally with my changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added documentation to describe my feature.
- [ ] I have squashed my commits into logic units.
- [x] I have described the changes in the commit messages.

### CODE REVIEW

File: docs/user/translating.rst [hunk 1]
```diff
@@ -393,3 +393,20 @@ The following operations are supported:
 .. seealso::
 
    :ref:`Bulk edit addon <addon-weblate.flags.bulk>`
+
+Matrix View
+------------------
```
Comment (nijel) [inline docs/user/translating.rst:398]:
```suggestion
Matrix View
-----------
```

File: docs/user/translating.rst [hunk 2]
```diff
@@ -393,3 +393,20 @@ The following operations are supported:
...
+
+To compare different languages efficiently you can use the matrix view. It is available on every component page under the :guilabel:`Tools` menu.
+First select all languages you want to compare and confirm your selection, after that you can click on any translation to open and edit it quickly.
+
+The matrix view is also a very good starting point to find missing translations in different languages and quickly add them from one view.
+
+Zen Mode
+------------------
```
Comment (nijel) [inline docs/user/translating.rst:406]:
```suggestion
Zen Mode
--------
```

**Review by nijel — COMMENTED** [review]
Looks good, I've made some minor suggestions on syntax.

**Comment by vabene1111** [comment]
thank you for the suggestions! i implemented the changes

**Comment by nijel** [comment]
Merged, thanks for your contribution!

**Pull request merged**